SB2018/2018-03-23.md

```diff
@@ -0,0 +1,14 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 16, they can fill out the forms themselves; however, if students are under 16, the form must be signed by their parent/guardian.
+
+These forms must be filled and sent to us to get a ticket for the competition, and to ultimately gain entry.
```

Comment:
Maybe set an explicit deadline here?

Reply:
fixed in 418d0e1f8b61c6eae68603410d8d44ae9e473615